node/pkg/allocateip/allocateip.go

```diff
@@ -89,7 +88,7 @@ func run(
 
 	if done == nil {
 		// Running in single shot mode, so assign addresses and exit.
-		reconcileTunnelAddrs(nodename, cfg, c, felixEnvConfig)
+		_ = reconcileTunnelAddrs(nodename, c, felixEnvConfig)
```

Comment:
```go
	defer func() {
		if retErr != nil {
			...
			log.Fatal("Failed to reconcile tunnel address")
		}
	}()
```

Ok, but the logs here won’t actually print because we are using Fatal in deferred function.

Reply:
Or we can remove Fatal in the deferred  function, and use Fatal here